builder/src/non_permissioned.rs

```diff
@@ -310,7 +310,7 @@ mod test {
         events_service_client.connect(None).await;
 
         let subscribed_events = events_service_client
-            .socket("hotshot-events/events")
+            .socket("hotshot-events/v1/events")
```

Comment:
v1 is the version which you get when the api is definied. If you don't hardcode this version, then the client would automatically redirect to the latest version. In this case, I think it would redirect to `v1/hotshot-events/events`